app/models/snpedia_paper.rb

```diff
@@ -9,4 +9,13 @@ class SnpediaPaper < ActiveRecord::Base
   def summary
     read_attribute(:summary).presence || "No summary provided."
   end
+
+  def matches_local_genotype?(other_local_genotype)
+    local_genotype == other_local_genotype.split('').sort.join
+  end
+
+  def local_genotype
+    url =~ /\((.*);(.*)\)$/
+    $1 + $2
+  end
```

Comment:
So, what about the name? Do you have a better suggestion or is this fine? Irrespective of the use case, what would you call the two letters in a SNPedia URL? On their pages it's just "genotype". Is this better than "local genotype"?